05_stringer/kasunfdo/main_test.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := "127.0.0.1"
+	actual := buf.String()
+
+	assert.Equalf(t, expected, actual, "Unexpected output")
+}
+
+func TestIPAddrStringer(t *testing.T) {
+	tests := map[string]struct {
+		input    IPAddr
+		expected string
+	}{
+		"Uninitialized IP": {
```

Comment:
Just a minor comment - I would call this "Zero value IP". Go does not have "uninitialized" values. They are all initialised to the type's zero value if not explicitly set to something else. No need to change this for this PR, just something to keep in mind.